internal/abci/abci.go

```diff
@@ -116,21 +119,26 @@ type AbciApp struct {
 
 	// Migrator is the migrator module that handles migrations
 	migrator MigratorModule
+
+	// lastCommitInfoFileName is the file name of the last commit info file
+	// which stores the app hash and height at the end of FinalizeBlock.
+	lastCommitInfoFileName string
 }
 
 func NewAbciApp(ctx context.Context, cfg *AbciConfig, snapshotter SnapshotModule, statesyncer StateSyncModule,
-	txRouter TxApp, consensusParams *chain.ConsensusParams, peers WhitelistPeersModule, migrator MigratorModule, db DB, log log.Logger) (*AbciApp, error) {
+	txRouter TxApp, consensusParams *chain.ConsensusParams, peers WhitelistPeersModule, migrator MigratorModule, db DB, dir string, log log.Logger) (*AbciApp, error) {
```

Comment:
I'm not sure think we need to or should add this `dir` param.  We already have a log of parameters, and the purpose and scope of this one is unclear from the function signuature.   Could be in `AbciConfig` or manage some other way?